deployment/migration-assistant-solution/lib/solutions-stack.ts

```diff
@@ -128,13 +128,13 @@ export class SolutionsInfrastructureStack extends Stack {
         bootstrapRole.addManagedPolicy(ManagedPolicy.fromAwsManagedPolicyName('AdministratorAccess'))
 
         new InstanceProfile(this, 'BootstrapInstanceProfile', {
-            instanceProfileName: `bootstrap-${stageParameter.valueAsString}-instance-profile`,
+            instanceProfileName: `bootstrap-instance-profile-${stackMarker}`,
             role: bootstrapRole
         })
 
         new Instance(this, 'BootstrapEC2Instance', {
             vpc: vpc,
-            instanceName: `bootstrap-${stageParameter.valueAsString}-instance`,
+            instanceName: `bootstrap-instance-${stackMarker}`,
```

Comment:
I'm not opposed to changing this naming pattern, but this definitely requires an update to our Solutions documentation which expect the previous naming pattern so just putting on your radar